src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -10,9 +11,31 @@
 
 namespace GitHub.ViewModels
 {
+    //Add properties to interface first
+    //Ex would be login tab view model
     [ExportViewModel(ViewType = UIViewType.PRCreation)]
     [PartCreationPolicy(CreationPolicy.NonShared)]
     class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationViewModel
     {
+        public PullRequestCreationViewModel(IRepositoryHost repositoryHost, ISimpleRepositoryModel repository)
```

Comment:
The initialization code is failing because this constructor is not marked `[ImportingConstructor]`, so MEF can't create an instance of this. Every class that's exported to MEF (marked with `[Export..]`) has to either have an empty public constructor or a constructor marked with `[ImportingConstructor]`, so that MEF knows which constructor to invoke when instantiating things.